tests/mips64/timer_tests.rs

```diff
@@ -0,0 +1,232 @@
+#[cfg(all(test, feature = "mips64_timer_tests"))]
+mod timer_tests {
+    use super::super::*;
+    use crate::timer::TickType;
+    use core::time::Duration;
+
+    #[test]
+    fn test_duration_to_ticks() {
```

Comment:
Needs tests description for each test. Sometimes it's not obviously, what is tested from test's name